### DISCUSSION TITLE
关于新版的文档有什么好的调试方法吗？

### DISCUSSION THREAD

**Discussion by KarasuShin** [DIYgod/RSSHub#15325] [Q&A] [answered]
新版本中将文档独立成了另一个仓库，在编写路由时没有办法随时查看路由说明文档的展示效果，容易遗漏一些排版问题。

我目前的调试方法是运行`npm run build:docs`，然后将`assets/build/docs`中的产物替换到本地的[rsshub-docs](https://github.com/RSSNext/rsshub-docs)项目中，但这样每次修改都要手动在两个项目目录中替换文件，有什么更便捷的方法吗？

**Comment by TonyRL** [comment, marked as answer]
You can use [symbolic link](https://linux.die.net/man/1/ln). 
```sh
ln -s /path/to/source/file /path/of/symbolic/link
```
